docs/guide/concept-autoloading.md

```diff
@@ -2,93 +2,68 @@ Class Autoloading
 =================
 
 Yii relies on the [class autoloading mechanism](http://www.php.net/manual/en/language.oop5.autoload.php)
-to locate and include all required class files. It provides a high-performance class autoloader that is compliant with the
+to locate and include all required class files. The loader itself is generated by Composer and complies to
 [PSR-4 standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-4-autoloader.md).
-The autoloader is installed when you include the `Yii.php` file.
 
 > Note: For simplicity of description, in this section we will only talk about autoloading of classes. However, keep in
   mind that the content we are describing here applies to autoloading of interfaces and traits as well.
-
-
-Using the Yii Autoloader <span id="using-yii-autoloader"></span>
-------------------------
-
-To make use of the Yii class autoloader, you should follow two simple rules when creating and naming your classes:
-
-* Each class must be under a [namespace](http://php.net/manual/en/language.namespaces.php) (e.g. `foo\bar\MyClass`)
-* Each class must be saved in an individual file whose path is determined by the following algorithm:
-
-```php
-// $className is a fully qualified class name without the leading backslash
-$classFile = Yii::getAlias('@' . str_replace('\\', '/', $className) . '.php');
+  
+  
+## Configuring autoloader <span id="configuring-autoloader"></span>
+
+Since Yii uses Composer-generated autoloader, in order to configure it, `composer.json` should be used:
+
+
+```json
+{
+    "name": "myorg/myapp",
+    "autoload": {
+        "psr-4": {
+          "app\\": "src",
+          "tests\\": "tests"
```

Comment:
`autoload-dev` for `tests`, see https://github.com/yiisoft/yii2/pull/14761#issuecomment-327290560